tests/test_unetr.py

```diff
@@ -123,7 +123,7 @@ def test_ill_arg(self):
             )
 
     @parameterized.expand(TEST_CASE_UNETR)
-    @SkipIfBeforePyTorchVersion((1, 9))
+    @SkipIfBeforePyTorchVersion((2, 0))
```

Comment:
The `SkipIfBeforePyTorchVersion` decorator has been modified to require PyTorch version 2.0 or later. However, it's not clear why this change was made

Reply:
Torchscript convert will fail since the new introduced flash attention.

Reply:
Shall we fallback to not using flash attention if version < 2.0?

Reply:
For example:
```
if use_flash_attention and not pytorch_after(minor=13, major=1, patch=0):
    raise ValueError(
        "use_flash_attention is only supported for PyTorch versions >= 2.0."
        "Upgrade your PyTorch or set the flag to False."
    )
```

to:

```
if use_flash_attention and not pytorch_after(minor=13, major=1, patch=0):
    log.warning(...)
    use_flash_attention = False
```

Reply:
Yes, it's true. But it may not influence torchscript convert which will go through all the sub-branch.
https://github.com/Project-MONAI/MONAI/blob/49a1e343129909f7e01c67882a71c12beed3ef30/monai/networks/blocks/selfattention.py#L90